### PR TITLE
Update research ultra-coarse-grained simulation

### PR DESCRIPTION
**Pull request by mzf0069**

### CODE REVIEW

File: _data/research.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 - title: Ultra-coarse-grained simulations of protein self-assembly
   subtitle: 
   group: featured
-  image: images/software.png
+  image: images/research-ultra-cg.png
   link:
```
Comment (clpetix) [inline _data/research.yaml:23]: Change this to `link: research/ultra-coarse-graining`

File: _data/research.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@
 - title: Ultra-coarse-grained simulations of protein self-assembly
   subtitle: 
   group: featured
-  image: images/software.png
+  image: images/research-ultra-cg.png
   link: 
   description:
```
Comment (clpetix) [inline _data/research.yaml:24]: Add a one line description of your research. You can see examples on me and Mayukh's pull request

Comment (clpetix) [reply _data/research.yaml:24]: Also, don't end this sentence with a punctuation mark to be consistent with Mayukh and me.

**Review by mayukh33 — APPROVED** [review]

**Review by clpetix — APPROVED** [review]

**Review by pgkovakas — APPROVED** [review]

**Comment by mayukh33** [comment]
Hi @mzf0069! Can you add this line at the end of your `index.md` file: `[See our publications!](../../publications/?search=surrogate+modeling)`? You can look at mine or Levi's `index.md` file for reference. You can change the search keyword (surrogate+modeling part) if you think some other keyword is applicable.

**Review by mayukh33 — APPROVED** [review]

**Comment by mphoward** [comment]
Thanks for this PR! I pushed a commit to reorganize it into the framework I implemented on Mayukh's PR.

Before merging, I would like to discuss some changes to the figure. Specifically, I would like to try to make it a little bit simpler. I'd also like to use Arial font to be consistent with Levi's figure. Let's discuss some ideas for that later!

**Comment by mphoward** [comment]
Also, please make sure to `git pull` before making any further changes yourself, since I pushed to your branch.

File: images/research-ultra-cg.png
Comment (mphoward) [inline images/research-ultra-cg.png:1]: This file should be removed.

**Review by mphoward — APPROVED** [review]
Thanks! I will remove the old `png` figure when I also lightly edit the text and merge. Thanks again!

**Pull request merged**